src/util/DeviceFingerprint.js

```diff
@@ -0,0 +1,79 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/*jshint latedef:false */
+/*global JSON */
+
+define(['vendor/lib/q', 'jquery'], function (Q, $) {
+
+  return {
+    generateDeviceFingerprint: function (oktaDomainUrl) {
+      var deferred = Q.defer();
+
+      // Create iframe
+      var $iframe = $('<iframe>', {
+        id: 'okta_fingerprint_iframe',
+        style: 'display: none;'
+      });
+      $iframe.appendTo('.auth-content');
```

Comment:
Along the same theme, we should pass in an el to this function to append to - i.e. we're currently assuming there are no other .auth-content divs on the page, but that might not be true. Better to have the views tell the function where to append to (in this case the primaryAuth view can just pass in it's el).